en-US/firefox-ios.xliff

```diff
@@ -132,6 +132,11 @@
         <target>Enter passcode</target>
         <note>Text displayed above the input field when changing the existing passcode</note>
       </trans-unit>
+      <trans-unit id="Face ID &amp; Passcode">
```

Comment:
@flodolo Can we get away with this `id` for now? I would like to convert all these strings to have a proper `New.Style.ID` after we ship v10.

Reply:
yes, we still have plenty of them